### PR TITLE
Convert project to Monorepo: Part 1

### PR DESCRIPTION
**Pull request by AnastasiiaPlutalova**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -15,6 +15,7 @@ jobs:
   test:
     name: "Tests"
     runs-on: ubuntu-latest
+    working-directory: test-app
```
Comment (SergeAstapov) [inline .github/workflows/ci.yml]: @AnastasiiaPlutalova minor, we don't need this line as we specify `working-directory` below for tasks

File: addon/package.json [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+  "scripts": {
+    "lint": "npm-run-all --aggregate-output --continue-on-error --parallel \"lint:!(fix)\"",
+    "lint:fix": "npm-run-all --aggregate-output --continue-on-error --parallel lint:*:fix",
+    "lint:hbs": "ember-template-lint .",
+    "lint:hbs:fix": "ember-template-lint . --fix",
+    "lint:js": "eslint . --cache",
+    "lint:js:fix": "eslint . --fix"
+  },
+  "dependencies": {
+    "babel-eslint": "^10.1.0",
```
Comment (SergeAstapov) [inline addon/package.json:33]: @AnastasiiaPlutalova all of the eslint, ember-template-lint, npm-run-all and prettier should be in devDependencies

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -15,6 +15,7 @@ jobs:
   test:
```
Comment (SergeAstapov) [inline .github/workflows/ci.yml]: @AnastasiiaPlutalova please adjust `floating` and `try-scenarios` as well to set `working-directory`

**Review by SergeAstapov — CHANGES_REQUESTED** [review]

File: .github/workflows/ci.yml (hunk 3 above)
Comment (AnastasiiaPlutalova) [reply .github/workflows/ci.yml]: @SergeAstapov added `working-directory: test-app` to floating and try-scenarios. Is `working-directory: addon` needed for them?

File: addon/package.json (hunk 2 above)
Comment (AnastasiiaPlutalova) [reply addon/package.json:33]: Done

File: .github/workflows/ci.yml (hunk 1 above)
Comment (AnastasiiaPlutalova) [reply .github/workflows/ci.yml]: Done

**Review by SergeAstapov — APPROVED** [review]

**Comment by villander** [comment]
@SergeAstapov @AnastasiiaPlutalova there's no CI running here, could you please fix that?

**Comment by SergeAstapov** [comment]
> First-time contributors need a maintainer to approve running workflows. 

@villander you need to approve PR to run CI :)

**Review by villander — APPROVED** [review]

**Pull request merged**